### PR TITLE
Added severity to CLI output

### PR DESCRIPTION
**Pull request by praxder**

### CODE REVIEW

**Comment by kuhnroyal** [comment]
I also added severity in https://github.com/invertase/dart_custom_lint/pull/86 which additionally adds JSON output.

**Comment by rrousselGit** [comment]
@praxder Do you mind fixing the conflicts?

This otherwise looks good to me. I'm happy with giving it a quick merge once that's fixed

**Comment by praxder** [comment]
@rrousselGit Updated. I had some trouble testing these changes locally, so I'm not confident that they work. Let me know if I need to change anything else. 😬 Long-term, it would be great to have a Contributors guide in this repo.

**Comment by rrousselGit** [comment]
LGTM! Assuming the tests pass, will merge :)

File: packages/custom_lint/lib/custom_lint.dart [hunk 1]
```diff
@@ -154,7 +154,7 @@ void _renderLints(
   for (final error in errors) {
     stdout.writeln(
       '  ${_relativeFilePath(error.location.file, workingDirectory)}:${error.location.startLine}:${error.location.startColumn}'
-      ' • ${error.message} • ${error.code}',
+      ' • ${error.message} • ${error.code} • ${lint.severity.name}',
```
Comment (rrousselGit) [inline packages/custom_lint/lib/custom_lint.dart:157]: The `lint` variable doesn't seem to exist. Maybe you're looking for the `error` variable?

**Comment by rrousselGit** [comment]
If you're looking at running custom_lint locally, one step is to change the `packages/custom_lint/tools/analyzer_plugin/pubspec.yaml` . See the TODO in there

**Comment by praxder** [comment]
@rrousselGit Updated. I think the one failing test was pre-existing, and introduced as part of this: https://github.com/invertase/dart_custom_lint/actions/runs/5277956808/jobs/9546659176

**Pull request merged**